client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/account/AccountHandler.java

```diff
@@ -44,11 +49,16 @@ private void handleGetUserResponse(GetUserResponse response) {
         Platform.runLater(() -> {
             if (response.isSuccess()) {
                 userInfo = response.getUser();
+                homeController.displayNickName();
```

Comment:
Maybe we should just include the nickname as a parameter rather than having the static variable.